clang-tools-extra/docs/ReleaseNotes.rst

```diff
@@ -91,6 +91,11 @@ Improvements to clang-query
 Improvements to clang-tidy
 --------------------------
 
+- It no longer processes declarations from system headers by default, greatly
```

Comment:
Would be nice to start with ":program:clang-tidy" (or something like that), as in this section we put also release notes that impact scripts.